package.json

```diff
@@ -46,68 +46,65 @@
   },
   "homepage": "https://github.com/GoogleChrome/workbox#readme",
   "devDependencies": {
-    "@google-cloud/storage": "^1.7.0",
-    "@octokit/rest": "^15.8.1",
+    "@babel/core": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@google-cloud/storage": "^2.0.3",
+    "@octokit/rest": "^15.12.0",
     "@std/esm": "^0.26.0",
-    "archiver": "^2.0.3",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.0.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.7.0",
+    "archiver": "^3.0.0",
+    "babel-eslint": "^9.0.0",
     "babylon": "^6.18.0",
-    "browser-sync": "^2.24.4",
     "bytes": "^3.0.0",
     "chai": "^4.1.2",
-    "chalk": "^2.1.0",
-    "chromedriver": "^2.38.3",
+    "chalk": "^2.4.1",
+    "chromedriver": "^2.42.0",
     "clear-require": "^2.0.0",
-    "comlinkjs": "^3.0.1",
+    "comlinkjs": "^3.0.3",
     "common-tags": "^1.8.0",
-    "copy-webpack-plugin": "^4.2.0",
-    "coveralls": "^3.0.1",
-    "depcheck": "^0.6.7",
-    "eslint": "^4.19.1",
-    "eslint-config-google": "^0.9.1",
-    "eslint-plugin-header": "^1.2.0",
-    "eslint-plugin-jsdoc": "^3.7.1",
+    "copy-webpack-plugin": "^4.5.2",
+    "coveralls": "^3.0.2",
+    "depcheck": "^0.6.11",
+    "eslint": "^5.6.0",
+    "eslint-config-google": "^0.10.0",
+    "eslint-plugin-header": "^2.0.0",
+    "eslint-plugin-jsdoc": "^3.8.0",
     "express": "^4.16.3",
-    "firebase-tools": "^3.18.5",
-    "fs-extra": "^6.0.1",
+    "firebase-tools": "^4.2.1",
+    "fs-extra": "^7.0.0",
     "geckodriver": "^1.12.2",
-    "glob": "^7.1.2",
+    "glob": "^7.1.3",
     "gulp": "^4.0.0",
-    "gulp-babel": "^7.0.0",
-    "gulp-rename": "^1.2.3",
-    "gulp-sourcemaps": "^2.6.1",
-    "gzip-size": "^4.0.0",
+    "gulp-babel": "^8.0.0",
+    "gulp-rename": "^1.4.0",
+    "gulp-sourcemaps": "^2.6.4",
+    "gzip-size": "^5.0.0",
     "html-webpack-plugin": "^3.2.0",
     "jsdoc": "^3.5.5",
     "jsdoc-baseline": "https://github.com/hegemonic/baseline/tarball/master",
-    "lerna": "^2.11.0",
+    "lerna": "^3.4.0",
     "minimist": "^1.2.0",
     "mocha": "^5.2.0",
-    "nyc": "^12.0.1",
+    "nyc": "^13.0.1",
     "pr-bot": "0.2.0",
     "pre-push": "^0.1.1",
-    "proxyquire": "^2.0.1",
+    "proxyquire": "^2.1.0",
     "require-dir": "^1.0.0",
-    "rollup": "^0.59.4",
-    "rollup-plugin-babel": "^3.0.2",
+    "rollup": "^0.66.2",
+    "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-uglify": "^4.0.0",
+    "rollup-plugin-terser": "^3.0.0",
     "rollup-stream": "^1.24.1",
     "selenium-assistant": "^5.3.0",
-    "semver": "^5.4.1",
+    "semver": "^5.5.1",
     "serve-index": "^1.9.1",
-    "service-worker-mock": "^1.7.2",
+    "service-worker-mock": "^1.9.3",
     "shelving-mock-indexeddb": "github:philipwalton/shelving-mock-indexeddb#d2c5e52b6f9903026f5bac31cbe758cbebd98661",
```

Comment:
@philipwalton as far as I can tell, the change in your fork never made it to the upstream project, so we're still pinned to this.

Reply:
Yeah, I'll file an issue to upstream that.